### ISSUE TITLE
Are the volumes have been non-rigid registered?

### ISSUE THREAD

**Issue by YMBrianHung** [adalca/medical-datasets#9]
I test the origin dice between the some 100 subjects and the first volume(OASIS_OAS1_0001_MR1.nii),  and the mean dice is 0.86, which is so high.
Are the volumes have been non-rigid registered?

**Comment by adalca** [comment]
They are not deformably registered. Which label files, and which labels, did you compute Dice over?

**Comment by YMBrianHung** [comment]
I compute the dice between #349 to #457 subjects' labels and the first(#1) subject's labels individually, and add up them, and compute the mean,  and the result is 0.86
```
DSC = []
for loop：
  fixed_label = sitk.GetArrayFromImage(sitk.ReadImage(os.path.join(args.label_dir, "OASIS_OAS1_0001_MR1_seg.nii.gz")))
  input_label = sitk.GetArrayFromImage(sitk.ReadImage(label_file))[np.newaxis, np.newaxis, ...]
  input_label = torch.from_numpy(input_label).to(device).float()
  dice = compute_label_dice(fixed_label, input_label[0, 0, ...].cpu().detach().numpy())
  DSC.append(dice)
print("mean(DSC): ", np.mean(DSC))
```
this is my code?
Is my code is error?

**Comment by adalca** [comment]
I guess I don't know what `compute_label_dice` does, but one reason why you might have artificially large  Dice is if you include the background label in the computation?

**Comment by YMBrianHung** [comment]
Thank you so much!!
I will recompute the Dice.
Do you know which interval should the Dice be in？

**Comment by adalca** [comment]
It depends over which labels, but I suspect that for aseg25 the average Dice over the anatomical structures should be around 60.

**Comment by YMBrianHung** [comment]
Thank you so much!!

Another question:
Is the origin dataset from the "OASIS-1: Cross-sectional MRI Data in Young, Middle Aged, Nondemented and Demented Older Adults" ?
I see the OASIS-1  contains 416 subjects. And your dataset contains 414

**Comment by adalca** [comment]
yes i think the dataset we released is all from oasis1

**Comment by YMBrianHung** [comment]
Do you lose 2 subject？

**Comment by adalca** [comment]
probably didnt pass QC at some point

**Comment by adalca** [comment]
I'm going to close this since I think the original question (are the data non-rigidly registered?) has been answered, but the conversation can stay open.
